githubClient.js

```diff
@@ -38,7 +38,7 @@ const uploadJsonFile = async (jsonString, fileName, day) => {
     if(!day){
       await uploadJsonFile(jsonString, fileName, getDay())
     }
-    return octokit.request(`PUT /repos/{owner}/{repo}/contents/bad-debt/${day || 'latest'}/{path}`, {
+    return octokit.request(`PUT /repos/{owner}/{repo}/contents/${REPO_PATH}/${day || 'latest'}/{path}`, {
```

Comment:
This was a miss from a previous feature I think, because without ${REPO_PATH} it would always send the .json into the bad-debt directory and not bad-debt-staging, even when activating the "staging mode"